website/docs/docs/dbt-versions/release-notes/71-Mar-2024/product-docs-jan-rn.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "January 2024 product docs updates"
+id: "product-docs-jan24"
+description: "Jan 2024: The Product docs team merged 110 PRs, made various updates to dbt, such as adding new Mesh FAQs, three new developer blog posts, new conversion metrics for Semantic Layer, and more."
+sidebar_label: "Update: Product docs changes"
+tags: [Jan-2024, product-docs]
+date: 2024-03-01
+sidebar_position: 09
+---
+
+Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! First, we’d like to thank the 10 new community contributors to docs.getdbt.com :pray: What a busy start to the year! We merged 110 PRs in January.
+
+Here's how we improved the [docs.getdbt.com](http://docs.getdbt.com/) experience:
+
+- Added new hover behavior for images and FAQs. This was subsequently modified to improve user experience.
```

Comment:
We might want to break this into two accomplishments since we walked back the hover behavior for FAQs
- Added new hover behavior for images.
- Added new expandables for FAQs